<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1398</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1401</width>
      <height>771</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Determination of color parameters</string>
     </attribute>
     <widget class="QLabel" name="label_AS">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>380</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Saturation amplitude :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Export">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>640</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Export color criterias</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_AT">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>340</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_V">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>120</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Green :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_H">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>210</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Hue :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_AH">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>340</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Hue amplitude :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pourcentages1">
      <property name="geometry">
       <rect>
        <x>1340</x>
        <y>380</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pourcentages1_2">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>250</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_AS">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>380</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_AL">
      <property name="geometry">
       <rect>
        <x>1170</x>
        <y>420</y>
        <width>160</width>
        <height>29</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_pourcentages2_2">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>290</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_AL">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>420</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Lightness amplitude :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_degres_2">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>210</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>°</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_pourcentages2">
      <property name="geometry">
       <rect>
        <x>1340</x>
        <y>420</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_R">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>80</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Red : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_S">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>250</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Saturation :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_degres">
      <property name="geometry">
       <rect>
        <x>1340</x>
        <y>340</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>°</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_L">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>290</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Lightness :</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_Image">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>961</width>
        <height>671</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_B">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>160</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Blue :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_SelectImage">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>700</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_value_R">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_value_G">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>120</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_value_B">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>160</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_value_H">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>210</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_value_S">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>250</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_value_L">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>290</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="showOriginalImage_checkBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>570</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Original Image</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Color segmentation of cloud</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_xml">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>80</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Select xml file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ply">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>150</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>15</horstretch>
        <verstretch>15</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Select ply file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>200</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Segment</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1398</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
